src/main/java/io/mosip/mimoto/dto/IssuerDTO.java

```diff
@@ -17,11 +17,11 @@
 @Data
 public class IssuerDTO {
     @Expose
```

Comment:
Don't commit these changes, as they are meant to be run locally with the modified issuer configuration file.

Reply:
okay

Reply:
@kamlesh012 this is still not reverted in this commit.

Reply:
Done. Reverted the commit.